postgresql-containers/bin/postgres-ha/pgbackrest/pgbackrest-archive-push-local-gcs.sh

```diff
@@ -27,15 +27,16 @@ CRUNCHY_DIR=${CRUNCHY_DIR:-'/opt/crunchy'}
 # Load the pgBackRest environmental variables
 source "${CRUNCHY_DIR}/bin/postgres-ha/pgbackrest/pgbackrest-set-env.sh"
 
+TIMEOUT=120
```

Comment:
is it possible to hit the timeout in case of a slow network? should we make it somehow configurable?